services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/settings.py

```diff
@@ -105,22 +128,6 @@ def update_service_params_from_settings(
                 # NOTE: The Docker REST API reads Reservation when actually it's Reservations
                 create_service_params["task_template"]["Resources"].update(param.value)
 
-        # publishing port on the ingress network.
-        elif param.name == "ports" and param.setting_type == "int":  # backward comp
-            create_service_params["labels"]["port"] = create_service_params["labels"][
-                "service_port"
-            ] = str(param.value)
-        # REST-API compatible
-        elif param.setting_type == "EndpointSpec":
-            if "Ports" in param.value:
-                if (
-                    isinstance(param.value["Ports"], list)
-                    and "TargetPort" in param.value["Ports"][0]
-                ):
-                    create_service_params["labels"]["port"] = create_service_params[
-                        "labels"
-                    ]["service_port"] = str(param.value["Ports"][0]["TargetPort"])
-
```

Comment:
Question: are these no longer useful? This was only used to extract the port aftewards?

Reply:
yes pretty weird he?